.chloggen/fix_43104.yaml

```diff
@@ -0,0 +1,31 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Maybe create two changelog entries: one for the bug fix (use max_fetch_size instead of default_fetch_size), and one for the deprecation. They're both word mentioning